data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parse/xml/ParseXmlProcessorConfig.java

```diff
@@ -72,6 +72,9 @@ public String getPointer() {
         return pointer;
     }
 
+    @Override
+    public int getDepth() { return 0; }
```

Comment:
Should there be validations that indicate this is not configurable for the other parse processors? Otherwise it will be a silent 0 even if configured right?

Reply:
There is `depth:` option in parse xml configuration.